pom.xml

```diff
@@ -27,9 +27,11 @@
     <module>phoenix-flume</module>
     <module>phoenix-pig</module>
     <module>phoenix-server-client</module>
-    <module>phoenix-server</module>
```

Comment:
I like the phoenix-queryserver and phoenix-queryserver-client as names of the modules. phoenix-server-client is hard to parse without knowing phoenix-server is phoenix query server.